src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -166,10 +166,10 @@
       [:div {}
        [comps/Button {:text "Sign In"
                       :href (str "/service/login?path="
-                                 (js/encodeURIComponent (let [hash (nav/get-hash-value)]
+                                 (js/encodeURIComponent (js/decodeURIComponent (let [hash (nav/get-hash-value)]
```

Comment:
That could be true - I didn't check that theory either. What I do know is that when you navigate to a workspace normally, the ":" gets encoded somewhere in the nav-context code (I couldn't figure out how to undo that encoding). So, whenever it is passed to something else, it is *already* encoded before traveling to any other service.